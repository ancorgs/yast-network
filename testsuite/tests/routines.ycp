{
    import "Testsuite";
    map READ = $[
        "target" : $[
            "tmpdir" : "/tmp",
            "stat" : $[],
        ],
    ];
    Testsuite::Init ([READ], 0);

include "testsuite.ycp";

include "network/routines.ycp";

Testsuite::Dump("list2items");
Testsuite::Test(``(list2items(["x","y"], 0)), [], nil);

Testsuite::Dump("hwlist2items");
Testsuite::Test(``(hwlist2items([$["name":"x"],$["name":"y"]], 0)), [], nil);
}
