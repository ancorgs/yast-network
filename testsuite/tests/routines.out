Read	.target.tmpdir "/tmp"
Read	.target.stat "/y2update/control.xml" $[]
Read	.target.stat "/control.xml" $[]
Read	.target.stat "/etc/YaST2/control.xml" $[]
Read	.target.stat "/usr/share/YaST2/control/control.xml" $[]
Log	No control file found
Log	control file missing
Dump	list2items
Return	[`item (`id (0), "x", true), `item (`id (1), "y", false)]
Dump	hwlist2items
Return	[`item (`id (0), "x", true), `item (`id (1), "y", false)]
